Layout_table.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import  Table, Column, Integer, String,Date,Time,Float
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+class table_structure(Base):
```

Comment:
There needs to be 2 separate tables:

-  for realtime bike availability of  bikepoints
- for historical bike availability of bikepoints 

With this structure, you'll only be able to store the realtime availability

Reply:
Thank you @dashu2410 sir  for the review  . I would submit the suggested update in two to three days.
Sir can you please give insight on  historical bike availability of bikepoints whether to store all previous data in single table or store data in separate table with update of realtime bike availabilty of bikepoint table.